flow360/component/simulation/migration/BETDisk.py

```diff
@@ -104,6 +111,10 @@ def _parse_flow360_bet_disk_dict(
     updated_bet_dict["omega"] = (
         updated_bet_dict["omega"] * u.rad / time_unit  # pylint: disable = no-member
     )
+
+    log.info("Provided temperature was used to calculate the value of omega in rad/s.")
+    log.info("Omega can be manually changed to a desired value and unit.")
```

Comment:
It won't be possible to change since the V0 Flow360.json has
```python
{
...
"omega":0.2
...
}
```
so there is not option for changing unit and also we want to divert user from hand writing JSON directly.

Reply:
The idea behind this info was to inform the user that after their BET was translated, they can manually set up the omega value and unit they want as showcased in example_bet_conversion.py
`"""
params.models[X].omega = 500 * fl.u.rpm
"""`

Reply:
I can delete line 116 which states that omega can be changed if you want

Reply:
I see what you mean. I thought you are asking user to change the JSON. Let's change the 
```suggestion
    log.info("You can print and correct the value and unit of `Omega` afterwards if needed.")
```
for less confusion.